### DISCUSSION TITLE
Allow a domain except one subdomain

### DISCUSSION THREAD

**Discussion by Jonathan-B-Reichel** [proginosko/LeechBlockNG#484] [Q&A]
how could I:

Block
blockme.keepme.com

but keep every other page at keepme.com?

My block set starts by blocking the whole internet with this:
*
**
and then If I have two relevant lines:
+*keepme.com*
blockme.keepme.com

it looks like the one with the '+' takes precedence.  Any way I can allow *keepme.com* EXCEPT blockme.keepme.com, which should remain blocked?

**Comment by proginosko** [comment]
You don't need both `*` and `**` to block all sites; `*` alone will do it. If you add `+keepme.com`, that will only allow keepme.com and **www.keepme.com** _unless_ you have selected the option "Block all subdomains (not just www)" under General options.

All this to say, the following should give you what you need:

`*`
`+keepme.com`

**Reply by Jonathan-B-Reichel** [reply]
Doesn't quite do the trick. Recall that I don't want to only keep keepme.com, I want every possible subdomain of keepme.com except one or two that I want to selectively block.

```
*
+keepme.com
```

It doesn't matter if I add keepme.com/blockme to the blockset like this:

```
*
+keepme.com
keepme.com/blockme
```

and whether or not I check the box you mentioned to "block all subdomains" in the General options, in every case it still allows keepme.com/blockme but I want this to be blocked.

I'm not sure how I can only block keepme.com/blockme but keep every other possible subdomain (for example keepme.com/keep1 and keepme.com/keep2, etc.)

Any idea?

**Comment by proginosko** [comment]
The easiest solution is to use two block sets.

Block Set 1:
`*`
`+keepme.com`

Block Set 2:
`keepme.com/blockme`

The neater but more complicated solution is to use a regular expression (see under Advanced Options).
